doc/modules/preprocessing.rst

```diff
@@ -74,6 +74,8 @@ dtype (unless specified otherwise):
 
 Some scaling pre-processors, such as :code:`whiten()` or :code:`zscore()`, will force the output to :code:`float32`.
```

Comment:
This is part of your problem @h-mayorquin? The forcing of the output? Just so I can stay on the same page.

Reply:
Yes, I don't know why we have that option but my proble most likely is ignorance. Maybe there are good scenarios that I jus't don't understand.

Reply:
I honestly don't know either, so I'm happy to learn too :)